c7n/resolver.py

```diff
@@ -166,20 +167,29 @@ def _get_values(self):
                 res = jmespath.search(self.data['expr'], data)
                 if res is None:
                     log.warning('ValueFrom filter: %s key returned None' % self.data['expr'])
-                return res
+                return set(res)
```

Comment:
Does it make sense to the `if instance of list check` here? In some cases, res could potentially be a string correct?